acoustid/indexclient.py

```
@@ -241,20 +242,20 @@ def close(self):
 
 
 class IndexClientPool(object):
-    def __init__(self, max_idle_clients=5, recycle=-1, **kwargs):
+    def __init__(self, max_idle_clients=5, recycle=-1, **kwargs) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `max_idle_clients`

(ANN001)